vite.config.ts

```diff
@@ -49,7 +49,7 @@ export default defineConfig({
     }
   },
   envDir: './',
-  envPrefix: 'REACT_',
+  //envPrefix: 'REACT_',
```

Comment:
can this be removed entirely?

Reply:
Though I had this commited, indeed it is not needed